### ISSUE TITLE
Broken symlinks

### ISSUE THREAD

**Issue by JulienPalard** [seawolf42/django-autometrics#1]
Hi, I spotted two broken symlinks in `sitepackages/`. I don't really get what you tried to achieve but it looks like you just need dependencies, which are already nicely documented in your `setup.py`, but they won't be installed in `../.env/` in my case, more than a `~/.venvs/django-autometrics/`, so... why versionning them and why addig `sitepackages` to your `os.path` manally?

**Comment by seawolf42** [comment]
I'm totally open to a better way to achieve want I want, but what I was going for here is solving the case where .env needs to be in a predictable place so that I can use it locally and also so that it works in Travis-CI... in the former case, I have a symlink from .env to $HOME/<my_virtualenvs>/<my_env_for_this_project> (since I'm using virtualenvwrapper), in the latter case, I make a symlink from .env to ~/virtualenv/python2.7/ (since that's where Travis-CI installs the virtualenv for the test run). If you have a better approach I'd love to learn what it is! :)

**Comment by JulienPalard** [comment]
venv relies on [sys.prefix](https://docs.python.org/3.5/library/sys.html?highlight=sys.prefix#sys.prefix) set by [site.py](https://docs.python.org/3.6/library/site.html) to find packages directory.

`site.py` does modify `sys.prefix` according to the path of the executed python. As, in en venv, the executed python will be *inside* the venv, `sys.prefix` will point to the venv, allowing to find packages in it:

```
$ python3 -m venv /tmp/venvs/licensename
$ /tmp/venvs/licensename/bin/python3 -c 'import sys; print(sys.prefix)'
/tmp/venvs/licensename
```

(Note I did not used *. bin/activate* to proove there is no environment variable tricks here, python is *just* using its path to deduce its prefix.)

Then, note than nothing more is needed as python knows everything needed to find a module installed in a venv:

```
$ /tmp/venvs/licensename/bin/python3 -m pip install licensename
Collecting licensename
  Using cached licensename-0.4.1-py3-none-any.whl
Collecting textunwrap (from licensename)
  Using cached textunwrap-0.1.1-py2.py3-none-any.whl
Installing collected packages: textunwrap, licensename
Successfully installed licensename-0.4.1 textunwrap-0.1.1
$ /tmp/venvs/licensename/bin/python3 -c 'import licensename; print(licensename.__version__)'
0.4
```

So, no need to modify `sys.path` manually at any point, nor it is needed to know where the venv is as Python knows it. If at one point you *really* need to know where your venv physically is, you still have access to `site.prefix` which will also cleanly tell you if you're ouside of a venv.

Hope that helps.

**Comment by seawolf42** [comment]
D'oh! But of course. I use the `sitepackages` trick on projects I deploy so that I can tightly control exactly which packages make it to App Engine, but that doesn't apply in this case. Agreed, it can come out. Removing that now.

**Comment by seawolf42** [comment]
Fixed with a17e19d9e3f91ce5f40b3dd54e92bcb88f528d7c
